yt_dlp/extractor/tver.py

```diff
@@ -38,10 +40,16 @@ def _real_initialize(self):
 
     def _real_extract(self, url):
         path, video_id = self._match_valid_url(url).groups()
-        main = self._download_json(
+        api_response = self._download_json(
             'https://api.tver.jp/v4/' + path, video_id,
-            query={'token': self._TOKEN})['main']
-        p_id = main['publisher_id']
+            query={'token': self._TOKEN})
+        main = api_response['main']
```

Comment:
shouldn't main be non-fatal too? since error message dont depend on it

Reply:
oops that shouldn't be fatal too

Reply:
inlined the variable in question